src/app/backend/nodestats.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/google/cadvisor/info/v1"
```

Comment:
Could you please use `goimports` on this file? It moves external imports (`github.com/google/cadvisor/info/v1`) below these coming directly from golang (for example `time`).